kernel/api/src/main/java/org/sakaiproject/user/api/CandidateDetailProvider.java

```diff
@@ -0,0 +1,67 @@
+package org.sakaiproject.user.api;
+
+import org.sakaiproject.site.api.Site;
+import org.sakaiproject.user.api.User;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * This is a provider interface that allows Assignments to provide addition details about candidates
+ * to the interface.
+ */
+public interface CandidateDetailProvider {
+	
+	public static final String USER_PROP_CANDIDATE_ID = "candidateID";
```

Comment:
These  constants aren't really related to the `CandidateDetailProvider` but are specific to some implementations, so do they need to be part of the API?